examples/userapi/02_apply.ipynb

```diff
@@ -246,14 +250,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "OOB detected due to time_M=2\n"
+      "OOB detected due to time_M=3\n"
```

Comment:
I haven't looked at the example code yet, but is this sane? 

IOW -- was there an issue with the prior implementation?